dev_guide/deployments.adoc

```diff
@@ -106,21 +122,26 @@ message and a new deployment will not be started.
 To get basic information about recent deployments:
```

Comment:
i feel we are mixing replication controllers and deployments here, we should make it consistent or explain on top what is "deployment" and what is "replication controller" and how are those two related together.